src/interfaces.py

```diff
@@ -15,26 +15,26 @@
 import devices.ph_probe as ph_probe  # pH
 import devices.ph_probe_mock as ph_probe_mock
 import devices.serial_mock as serial_mock
-import devices.temp_probe as temp_probe
-import devices.temp_probe_mock as temp_probe_mock
-import devices.tempcontrol as tempcontrol  # Temp
-import devices.tempcontrol_mock as tempcontrol_mock
+import devices.temperature_probe as temperature_probe
+import devices.temperature_probe_mock as temperature_probe_mock
+import devices.temperaturecontrol as temperaturecontrol  # Temperature
+import devices.temperaturecontrol_mock as temperaturecontrol_mock
```

Comment:
You correctly changed the filenames to `temperature_control` and `temperature_control_mock` however these imports need to match that for them to be able to import